insightsengineering/teal.modules.general
# name reference fix

closes #323 

to test:

```r
library(scda)
devtools::load_all()

ADSL <- synthetic_cdisc_data("latest")$adsl

app <- teal::init(
  data = cdisc_data(cdisc_dataset(
    "ADSL", ADSL,
    code = "ADSL <- synthetic_cdisc_data(\"latest\")$adsl"
  ),
  check = TRUE
  ),
  modules = root_modules(
    tm_a_pca(
      "PCA",
      data_extract_spec(
        dataname = "ADSL",
        select = select_spec(
          choices = variable_choices(data = ADSL, c("BMRKR1", "AGE", "EOSDY")),
          selected = c("BMRKR1", "AGE"),
          multiple = TRUE
        ),
        filter = NULL
      )
    )
  )
)

shinyApp(app$ui, app$server)
```

![Screen Shot 2022-01-24 at 9 58 39 AM](https://user-images.githubusercontent.com/12874196/150818360-0241cc60-abaa-4e76-8f14-ff14b22da83a.png)
